apps/coscrad-frontend/cypress/support/commands.ts

```diff
@@ -32,9 +32,9 @@ Cypress.Commands.add('mount', mount);
 
 //
 // -- This is a parent command --
-Cypress.Commands.add('login', (email, password) => {
-    console.log('Custom command example: Login', email, password);
-});
+// Cypress.Commands.add('login', (email, password) => {
```

Comment:
remove commented code

Reply:
Maybe that will be the time to remove unused projects \ libs, too.